google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -761,4 +762,31 @@ def do_rollback(self, dbapi_connection):
         ):
             pass
         else:
-            dbapi_connection.rollback()
+            trace_attributes = {"db.instance": dbapi_connection.database.name}
+            with trace_call("SpannerSqlAlchemy.Rollback", trace_attributes):
+                dbapi_connection.rollback()
+
+    def do_commit(self, dbapi_connection):
+        trace_attributes = {"db.instance": dbapi_connection.database.name}
+        with trace_call("SpannerSqlAlchemy.Commit", trace_attributes):
+            dbapi_connection.commit()
+
+    def do_close(self, dbapi_connection):
+        trace_attributes = {"db.instance": dbapi_connection.database.name}
+        with trace_call("SpannerSqlAlchemy.Close", trace_attributes):
+            dbapi_connection.close()
+
+    def do_executemany(self, cursor, statement, parameters, context=None):
+        trace_attributes = {"db.statement": statement, "db.params": parameters}
```

Comment:
Please also include the `db.instance` attribute here and in the methods below. You should be able to get the database name via `cursor.connection.database.name`.  This makes the span much more useful as it tells the user which database the statement was run against.